### PR TITLE
User Management backend - Base - DB Models

### PR DESCRIPTION
**Pull request by ahinoamta**
Added DB models- Token model, and extended User model

@DoRTaL94 @RoniRush

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -16,3 +16,6 @@ _pycache_/
 
 # Python
 *.pyc
+*.orig
+*.pyc.*
```
Comment (simzacks) [inline .gitignore:20]: wouldn't this be handled by the *.pyc above?
I made this comment already in PR #46  so it looks like these have to be merged in order.

**Review by simzacks — COMMENTED** [review]
Other than the small comment about the .gitignore, it looks ready to go

File: .gitignore (hunk 1 above)
Comment (ahinoamta) [reply .gitignore:20]: No, becasue the files look like this:
conftest.cpython-36-pytest-5.4.2.pyc.1764

When I try to remove '*.pyc.*', all the files in the pattern above were unstaged in Git.

Comment (ahinoamta) [reply .gitignore:20]: We will be glad that this PR will be merged before #46 😃

**Review by Omeramsc — APPROVED** [review]
LGTM

**Pull request merged**